src/components/cardLT/StepsMission/planetSteps.d.ts

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+
+///-----------------------------------------
+/// Interface do Componente
+interface IStepParams {
+    /**
+     * @prop {number} step: Step (Marte, Júpter, Saturno, Urano, Netuno)
+     */
+     step: number
+    /**
+     * @prop {string} status: Status Planeta (B- Bloqueado, I- Inativo e A- Ativo)
+     */
+     status : string
+     idioma? : 'pt-BR' | 'pt-PT' | 'en-US' | 'es'   
```

Comment:
Acredito que seria melhor usar a palavra lang pra esse tipo de props, mas se for dar mt trabalho esquece por agr.